### PR TITLE
Enhancements to support ReactNative

### PR DESCRIPTION
**Pull request by toddsouthenbentley**

### CODE REVIEW

File: mobile-sdk/src/main/java/com/github/itwin/mobilesdk/ITMApplication.kt [hunk 1]
```diff
@@ -357,8 +357,9 @@ abstract class ITMApplication(
      *
      * @param context The [Context].
      * @param allowInspectBackend Allow inspection of the backend code, default false.
+     * @param existingWebView An existing webview to use instead of creating one. Note:
```
Comment (tcobbs-bentley) [inline mobile-sdk/src/main/java/com/github/itwin/mobilesdk/ITMApplication.kt:360]: Note what?

Comment (tcobbs-bentley) [reply mobile-sdk/src/main/java/com/github/itwin/mobilesdk/ITMApplication.kt:360]: Also, we should probably state that they need to override the web view's onConfigurationChanged to call ITMApplication's onConfigurationChanged.

Comment (tcobbs-bentley) [reply mobile-sdk/src/main/java/com/github/itwin/mobilesdk/ITMApplication.kt:360]: Maybe overscrollBy, scrollTo, and computeScroll, although I'm not sure what those do right now in ours.

Comment (toddsouthenbentley) [reply mobile-sdk/src/main/java/com/github/itwin/mobilesdk/ITMApplication.kt:360]: They don't have to override the web view's onConfigurationChanged, the app's activity can do this instead. We could consider removing that override from our code. In the case of ReactNative, we can't override any of the web view's methods (without creating our own React Native web view sub-class). Right now, I'm turning off overscroll in the web view and that seems to be sufficient.

I removed the "Note:" from the comment.

**Review by tcobbs-bentley — APPROVED** [review]

**Pull request merged**